apps/condo/domains/common/components/settings/MobileFeatureConfigContent.tsx

```diff
@@ -0,0 +1,55 @@
+import { Col, Row } from 'antd'
+import { Gutter } from 'antd/es/grid/row'
+import get from 'lodash/get'
+import React  from 'react'
+
+import { useFeatureFlags } from '@open-condo/featureflags/FeatureFlagsContext'
+import { useOrganization } from '@open-condo/next/organization'
+
+import { CardsContainer } from '@condo/domains/common/components/Card/CardsContainer'
+import { SettingCardSkeleton } from '@condo/domains/common/components/settings/SettingCard'
+import {
+    OnlyProgressionMeterReadingsSettingCard,
+} from '@condo/domains/settings/components/ticketSubmitting/OnlyProgressionMeterReadingsSettingCard'
+import { TicketSubmittingSettingCard } from '@condo/domains/settings/components/ticketSubmitting/TicketSubmittingSettingCard'
+import { MOBILE_FEATURE_CONFIGURATION, TICKET_SUBMITTING_SETTING, ONLY_PROGRESSION_METER_READINGS_SETTING } from '@condo/domains/settings/constants'
+import { MobileFeatureConfig as MobileFeatureConfigAPI } from '@condo/domains/settings/utils/clientSchema'
+
+const CONTENT_GUTTER: Gutter | [Gutter, Gutter] = [0, 40]
+
+export const MobileFeatureConfigContent: React.FC = () => {
+    const userOrganization = useOrganization()
+    const userOrganizationId = get(userOrganization, ['organization', 'id'])
+    const { obj: mobileConfig, loading } = MobileFeatureConfigAPI.useObject({
+        where: {
+            organization: { id: userOrganizationId },
+        },
+    })
+    const { useFlag } = useFeatureFlags()
+    const hasMobileFeatureConfigurationFeature = useFlag(MOBILE_FEATURE_CONFIGURATION)
+    const hasTicketSubmittingSettingFeature = useFlag(TICKET_SUBMITTING_SETTING)
+    const hasOnlyProgressionMeterReadingsSettingFeature = useFlag(ONLY_PROGRESSION_METER_READINGS_SETTING)
```

Comment:
Is every feature controlled by two feature flags combination?

Reply:
Yes, this is a business requirement. It is necessary that the org manages this itself. But at the same time, we do not want every user to see all the settings. Since we are developing them, not so that they would be turned off, but so that they would be used. This task is a rare case when it is necessary